src/dftbp/wavegrid/basis/spharmonics.F90

```diff
@@ -0,0 +1,135 @@
+!--------------------------------------------------------------------------------------------------!
+!  DFTB+: general package for performing fast atomistic simulations                                !
+!  Copyright (C) 2006 - 2025  DFTB+ developers group                                               !
+!                                                                                                  !
+!  See the LICENSE file for terms of usage and distribution.                                       !
+!--------------------------------------------------------------------------------------------------!
+
+#:include 'common.fypp'
+
+!> Real spherical harmonics up to l=4.
+!! See also: spharmonics.cuh
+module dftbp_wavegrid_basis_spharmonics
+  use dftbp_common_accuracy, only : dp
+  implicit none
+
+  private
+
+  public :: realTessY
+
+  ! Normalisation Constants labeled C_l_m
+  ! l = 0
+  real(dp), parameter :: C0_0 = 0.28209479177387814_dp ! 1/2 * sqrt(1/pi)
```

Comment:
Maybe, we should let the compiler to calculate it in the default precision, e.g.
```suggestion
  real(dp), parameter :: C0_0 = 0.5_dp * sqrt(1.0_dp / pi)
```
etc.